services/audio_library.py

```diff
@@ -0,0 +1,175 @@
+import asyncio
+import threading
+from os import path, listdir
+
+from api.interface import AudioFile, AudioFileConfig
+from services.audio_player import AudioPlayer
+from services.file import get_writable_dir
+
+class AudioLibrary:
+    def __init__(
+        self,
+        callback_playback_started: callable|None = None,
+        callback_playback_finished: callable|None = None
+    ):
+        # Configurable settings
+        self.callback_playback_started = callback_playback_started # Parameters: AudioFileConfig, AudioPlayer, volume(float)
+        self.callback_playback_finished = callback_playback_finished # Parameters: AudioFileConfig
+        
+        # Internal settings
+        self.audio_library_path = get_writable_dir("audio_library")
```

Comment:
make the string a constant so that it could be imported from another class (see `ConfigManager` etc.)

Reply:
Moved to constant